tests/pytests/unit/modules/test_at.py

```diff
@@ -0,0 +1,238 @@
+"""
+    :codeauthor: Rupesh Tare <rupesht@saltstack.com>
+
+    TestCase for the salt.modules.at module
+"""
+
+
+import pytest
+
+import salt.modules.at as at
+import salt.utils.path
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {at: {}}
+
+
+@pytest.fixture
+def atq_output():
+    return {
+        "jobs": [
+            {
+                "date": "2014-12-11",
+                "job": 101,
+                "queue": "A",
+                "tag": "",
+                "time": "19:48:47",
+                "user": "B",
+            }
+        ]
+    }
+
+
+@classmethod
```

Comment:
Sorry I didn't see this before, but I don't think this classmethod is required,, or would even be getting called now that its in pytest.

Reply:
Yes, it doesnt seem to be needed. Removed it.